src/loaders/FileLoader.js

```diff
@@ -173,7 +179,29 @@ class FileLoader extends Loader {
 
 					default:
 
-						return response.text();
+						return response.arrayBuffer().then( ab => {
+
+							let label = 'utf-8';
+							if ( this.mimeType !== undefined ) {
```

Comment:
Can you please explain a bit the purpose of this code section?

Reply:
Set default utfLabel for `new TextDecoder( utfLabel )`

Reply:
So this code is about extracting the `utfLabel` for `TextDecoder`'s constructor.